src/library/Table/themes.js

```diff
@@ -40,9 +40,9 @@ export const tableCellTheme: TableCellThemeFn = (baseTheme) => ({
 });
 
 export const tableHeaderTheme: TableHeaderThemeFn = (baseTheme) => ({
-  TableHeader_borderBottom: `2px solid ${baseTheme.borderColor}`,
-  TableHeader_borderBottom_highContrast: `2px solid ${baseTheme.color_gray_80}`,
-  TableHeader_borderTop: `1px solid ${baseTheme.borderColor}`,
+  TableHeader_borderBottom: `1px solid ${baseTheme.borderColor}`,
```

Comment:
looking for feedback here @tx-steven . Not sure if this is the best place to change this. Since this is hardcoded, I'm assuming this is where I should make this change.

Reply:
Pretty sure this is the right place for this one. Usually if they have a hardcoded value. We will just change the hardcoded value. Since this is so specific to the TableHeader and nothing else. It wont affect any other components in the library. So I think this is ok to do here.

Reply:
👍